pkg/action/cl001/ac011/explainer.go

```diff
@@ -5,5 +5,6 @@ import (
 )
 
 func (e *Explainer) explain(ctx context.Context) (string, error) {
-	return "", nil
+	s := `Cleanup kiam test resources from the tenant cluster`
```

Comment:
Just some food for thought. I see this as a wikipedia article. The explainer should not just drop some one liner but rather explain the circumstances and requirements of the action itself. Reading this one should get some pointers how to move on. I find this line as such not very helpful. Though I leave it to you to decide what is good enough here.